src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -38,29 +38,90 @@ To provide a good user experience, **sites should strive to have an LCP of 2.5 s
+
+The LCP for the whole origin may be very different to an individual page's LCP depending on how the LCP is loaded on that page compared to other pages on that origin. It can also be affected by how visitors navigate to these pages. Home pages tend to be visted by new users and so may often be loaded "cold", without any cached content and so are often the slowest pages on a website.
+
+Looking at the four difference pieces of CrUX data can help you understand whether an LCP issue is specific to this page, or a more general site-wide issue. Similarly, it can show which device types have LCP issues.
+
+### Using PageSpeed Insights CrUX supplementary metrics
+
+As well as LCP, those looking to optimize LCP, should also use the [First Contentful Paint (FCP)](/fcp/) and [Time to First Byte (TTFB)](/ttfb/) timings which can provide valuable insights into LCP.
+
+TTFB is the time when the visitor starting to navigate to a page (for example, clicking on a link), until the first bytes of the HTML document are received. A high TTFB can make achieving a 2.5 second LCP challenging, or even impossible.
+
+A high TTFB can be due to mutiple server redirects, visitors located far away from the nearest site server, visitors on poor network conditions, or an inability to use cached content due to query parameters.
+
+Once a page starts rendering there may be an initial paint (for example, the background color is painted), followed by some content showing up (for example, the site header). The appearnce of the initial conent is measured by FCP. The delta between FCP and other metrics can be very telling.
+
+A large delta between TTFB and FCP could indicate that the browser needs to download a lot of render-blocking assets. It can also be a sign it must complete a lot of work to render any meaningful content—a classic sign of a Client Side Rendered site.
+
+A large delta between FCP and LCP indicates that the LCP resource is either not immediately available for the browser to prioritise (for example, text or images that are managed by JavaScript rather than being available in the initial HTML) or that the browser is completing other work before it can display the LCP content.
+
+### Using PageSpeed Insights Lighthouse data
+
+The Lighthouse section of PageSpeed Insights offers some guidance to improving LCP, but first you should check if the LCP given is broadly inline with real user data provided by CrUX.
+
+If Lighthouse is showing no LCP issue, but the CrUX data is, then any Lighthouse suggestions may not be relevant. The opposite is also true—if Lighthouse is showing a really poor LCP time, but CrUX data is showing your users mostly have a good LCP, then it may not be necessary to further improve LCP. Just be sure to check that the CrUX data is for this page and not for the full origin as detailed above.
```

Comment:
Hm, I generally agree with this, _but_—if CrUX data shows a good LCP, but there are Lighthouse audits that give developers actionable information to make LCP potentially faster, would we want to instruct developers ignore those opportunities if the impact could be substantial, even for a website with "good" LCP?

Reply:
That's why I said "may not", but maybe that's too subtle.

WDYT about:

```suggestion
If Lighthouse is showing no LCP issue, but the CrUX data is, then any Lighthouse suggestions may not be relevant. The opposite is also true—if Lighthouse is showing a really poor LCP time, but CrUX data is showing your users mostly have a good LCP, then you may wish to consider the priority of optimizing LCP further, or if time is better spent on other performance improvements. Also be sure to check that the CrUX data is for this page and not for the full origin as detailed above.
```